dataflow/manual/content.tex

```diff
@@ -1764,17 +1681,13 @@ \section{Example: Live Variable}
 is a backward transfer function). The transfer function visits assignments to
 update the information of live variables for each node in the stores.
 
-\textbf{Example.} An example is shown in \autoref{lst:LiveSimple} and
-\autoref{fig:LiveSimple}.
+\textbf{Example.} An example is shown in \autoref{fig:LiveSimple}.
 
-\livevariable{Simple}{Simple sequential program to illustrate live variable.}
+\flow{LiveSimple}{.33}{1}{Simple sequential program to illustrate live variable.  Intermediate analysis results are shown.}
 
 
 \section{Default Analysis}
 
-\begin{workinprogress}
-I feel like there is a missing cross-reference or two to this section.
```

Comment:
Is it worth keeping this at least as comment? It doesn't look like you added a reference.

Reply:
I didn't see anywhere that a cross-reference was missing.  If we notice one in the future, we can add it.